boards/arm/nrf9160_pca10090/nrf9160_pca10090ns.yaml

```diff
@@ -0,0 +1,7 @@
+identifier: nrf9160_pca10090ns
```

Comment:
Should we add the RAM and FLASH properties?
`ram: 256 flash: 1024`

Reply:
Same as above. I will add them.

Reply:
@lemrey @SebastianBoe , what values, exactly, do we need to add here?
- I added RAM=64K and FLASH=256K, because _this_ is what the current board definition allows by default. These are different from the SOCs actual resources, though

Reply:
It is used to filter tests that require a lot of ram.

So it must match what the board definition allows.

Reply:
Then this is fine.